src/test/codegen/bpf-alu32.rs

```diff
@@ -0,0 +1,12 @@
+// compile-flags: --emit=asm --target bpfel-unknown-none
```

Comment:
Shouldn't this be `--emit=llvm-ir`?

Reply:
Oops yes that's right, I started the test as an asm test and then moved it over. I've removed the whole line since it isn't really needed anymore.